src/common.jl

```diff
@@ -21,6 +21,13 @@ struct Rect
     top::Float64
 end
 
+union(r1::Rect, r2::Rect) =
```

Comment:
Maybe TODO for using Extents?

Reply:
`Rect` is the lower lever Shapefile object for only x/y extent, and we need to write it quite often so it was worth having this so we can write the union of all the shape level rects as the file level rect.